examples/access-controllable-examples/README.md

```diff
@@ -0,0 +1,274 @@
+# Example of using the Access Control plugin
+
+An access control mechanism that allows you to specify which groups of users can access certain functions.
+
+```rust
+use near_plugins::AccessControllable;
+use near_plugins::AccessControlRole;
+use near_plugins_derive::access_control;
+use near_plugins_derive::access_control_any;
+use near_sdk::near_bindgen;
+use borsh::{BorshSerialize, BorshDeserialize};
+use near_plugins::events::AsEvent;
```

Comment:
```suggestion
```

This shouldn't be required anymore (ref. #21).

Reply:
fixed https://github.com/aurora-is-near/near-plugins/pull/34/commits/a334a44069b91318440e1a87c74f219a248b9920